src/app/routes/onDemandTV/getInitialData/index.js

```diff
@@ -0,0 +1,83 @@
+import path from 'ramda/src/path';
```

Comment:
A lot of this is duplicated with the Radio brand - I'd recommend either:

- Removing all the unused stuff from here, and adding it back in as we need it when we're building out the page
- Or leaving it like this, but having both TV and Radio pages use the same `getInitialData` function for now (if possible)

Reply:
Yeah my plan was to use the same function for both but I don't yet know what data is required for onDemand TV. I'll strip it back and we can build it back up when we come to adding components to the page.